From db8591d5f5e6d717d45c1b2e2b3d04d26bda3297 Mon Sep 17 00:00:00 2001
From: Liang Li <liang.li@windriver.com>
Date: Thu, 3 Jun 2010 09:05:19 +0800
Subject: [PATCH] wrhv/p4080ds: fix trace_irq build error

Due to commit 37ae8ebd66e1a99d8ca4a935695ff63f2d9ad5eb, the
trace_irq enter/exit was disabled. So delete them here.

Signed-off-by: Liang Li <liang.li@windriver.com>
---
 arch/powerpc/kernel/vbi/wrhv.c |    8 --------
 1 files changed, 0 insertions(+), 8 deletions(-)

diff --git a/arch/powerpc/kernel/vbi/wrhv.c b/arch/powerpc/kernel/vbi/wrhv.c
index 4bc8f89..dde98d4 100644
--- a/arch/powerpc/kernel/vbi/wrhv.c
+++ b/arch/powerpc/kernel/vbi/wrhv.c
@@ -701,8 +701,6 @@ void wrhv_do_direct_IRQ(struct pt_regs *regs)
 	struct pt_regs *old_regs = set_irq_regs(regs);
 	unsigned int irq;
 
-	trace_irq_entry(regs);
-
 	irq_enter();
 
 	check_stack_overflow();
@@ -726,8 +724,6 @@ void wrhv_do_direct_IRQ(struct pt_regs *regs)
 	}
 #endif
 
-	trace_irq_exit(regs);
-
 }
 
 void wrhv_do_IRQ(struct pt_regs *regs)
@@ -735,8 +731,6 @@ void wrhv_do_IRQ(struct pt_regs *regs)
 	struct pt_regs *old_regs = set_irq_regs(regs);
 	unsigned int irq;
 
-	trace_irq_entry(regs);
-
 	irq_enter();
 
 	check_stack_overflow();
@@ -761,8 +755,6 @@ check_again:
 		timer_interrupt(regs);
 	}
 #endif
-
-	trace_irq_exit(regs);
 }
 
 unsigned int wrhv_map_irq_of_desc(char *irq_name, int32_t irq_dir)
-- 
1.6.5.2

